.pre-commit-config.yaml

```diff
@@ -28,6 +28,12 @@ repos:
       - id: isort
         args: [--settings-path, ./pyproject.toml]
 
+  - repo: https://github.com/pre-commit/mirrors-yapf
+    rev: v0.30.0
+    hooks:
+      - id: yapf
+        args: [--parallel, --in-place]
```

Comment:
shall it be also recursive?

Reply:
I don't think so, see my previous answer: https://github.com/PyTorchLightning/pytorch-lightning/pull/5747#discussion_r568580819